spynnaker7/pyNN/models/pynn_projection.py

```diff
@@ -27,7 +32,11 @@ def __init__(
             synapse_dynamics=None, rng=None):
         # pylint: disable=too-many-arguments, protected-access
         synapse_dynamics_stdp = None
-        if synapse_dynamics is None:
+        if synapse_dynamics is None or \
```

Comment:
if ya put a ( at beginning of if, then you can remove the \  (assuming ) at end of blah)